assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -166,13 +172,13 @@ const ChecklistTab = ({
             pages: {},
           }
         ),
-    [checklist, enablePrePublishVideoOptimization]
+    [checklist, canOptimizeVideo]
   );
 
   const getOnPrepublishSelect = useCallback(
     (args) => {
-      const { elements, elementId, pageId, highlight } = args;
-      if (!elements && !elementId && !pageId && !highlight) {
+      const { highlight } = args;
```

Comment:
@embarks is this ok? It felt like overkill? We only want to enable the arrow to click to the element when highlight exists, yes? Now with video optimization the `elements` and `elementId` exists for that but it shouldn't have the arrow.

Reply:
should be fine! I don't believe any of those are accessed that would cause bugs, and tests should catch any

Reply:
_arrested development narrator voice_:  it was not fine. 

Turns out these checks are important! I went ahead and just added an escape hatch now that some checks are getting more complicated actions. It's called `noHighlight`. Now tests pass!